### DISCUSSION TITLE
Dashboard only partly working

### DISCUSSION THREAD

**Discussion by brslt** [kubeshop/kusk-gateway#652] [Q&A] [answered]
I decided to try Kusk today, and installed it from my Windows workstation onto a Azure AKS cluster. No errors during install, and services are running in the kusk-system namespace. I deployed a couple of dummy APIs in the Dashboard, but here, the navigation is lacking a lot. Some examples:
- Paths/add a new route: no response
- Dashboard, Deployments, Analytics: blank UI
- Settings: Basically, no changes are saved in any submenu
- Even Deleting the API doesn't work, gives error message "Something went wrong"

Accessing the API through the load balancer gives a 404 error.

Is there something fundamentally wrong with the installation? Thanks for any feedback.

**Comment by jasmingacic** [comment]
Can you please provide us with the following:

Kusk version that you used "kusk version"
Host CPU  architecture amd or arm.

And if possible log output from deployment kusk-system/kusk-gateway

If you have any other observations please share it with us.

Once we get more info we will take a look and get it fixed.

**Comment by povilasv** [comment, marked as answer]
Hey @torbjorn-norgesbuss, sorry for the issue. It turns out we accidentally released the work in progress version of our dashboard, so that's why it's broken for you.

I just fixed it by doing a new release with the old version https://github.com/kubeshop/kusk-gateway-dashboard/releases/tag/v1.1.0-stable

To get the fixed version you need to upgrade your kusk gateway installation using:
```
kusk upgrade
```

**Reply by brslt** [reply]
Aha! Great. I can agree the whole thing looked a bit broken... Will do the upgrade asap.

**Reply by kylehodgetts** [reply]
Thanks for your patience @torbjorn-norgesbuss :)

**Reply by brslt** [reply]
Which Dashboard version should be used? When trying to upgrade now, first the wholde process hanged and stopped when downloading 1.1.1-rc5, then I tried again, now it's trying with 1.1.0-stable

**Reply by brslt** [reply]
And actually, both times it timed out with this message:
     ERROR:  Looking for Helm: process error: exit status 1
             output: Error: UPGRADE FAILED: timed out waiting for the condition

I could of course try a clean install instead. Will try that when back from meetings later today.

**Reply by kylehodgetts** [reply]
Would you be happy to hop into our discord so we can debug this with you?
https://discord.com/invite/6zupCZFQbe

**Reply by brslt** [reply]
Yep, I'm in

**Reply by jasmingacic** [reply]
What's your nickname in discord?

**Comment by brslt** [comment]
"barslett". I just posted in the general thread.

Få Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by jasmingacic** [reply]
I've pinged you in this channel
https://discord.com/channels/884464549347074049/913784323075866624

**Comment by brslt** [comment]
After some problems with upgrading after this incident, I installed Kusk on a clean cluster, and the dashboard now seems to work as supposed. I will redeploy myu own services and continue my testing. Will let you know on Discord if I bump into anything else!

**Reply by kylehodgetts** [reply]
That's great news!
